chart/kafka-ui/README.md

```diff
@@ -0,0 +1,22 @@
+# Kafka-UI Helm Chart
+## Configuration
+Most of the Helm charts parameters are common, follow table describe unique parameters related to application configuration.
+### Kafka-UI parameters
+| Parameter| Description| Default|
+|---|---|---|
+| `existingConfigMap`| Name of the existing ConfigMap with Kafka-UI environment variables | `nil`|
+| `existingSecret`| Name of the existing Secret with Kafka-UI environment variables| `nil`|
+| `envs.secret`| Set of the sensitive environment variables to pass to Kafka-UI | `{}`|
+| `envs.normal`| Set of the environment variables to pass to Kafka-UI | `{}`|
+
+## Example
+To bootstrap minimal environment with Apache Kafka and Kafka-UI need to execute follow:
+``` bash 
```

Comment:
do we need instruct to `cd example` before helm commands?

Reply:
`example` removed from this repo